### PR TITLE
feat(icons): add IconQuoteLarge

### PR DESCRIPTION
**Pull request by kaloster**
Add IconQuoteLarge Large and X-Large

## Summary

**Icon**
Quote Icon for [Citation Tool](https://github.com/chanzuckerberg/single-cell-data-portal/pull/5836)
 
**Component Name:** IconQuoteLarge

**Component Requirements:** 
**Icons can be found on the Google Drive:**
- [x] [IconQuoteLarge](https://drive.google.com/file/d/1CNQAqudoOr1Ukhx5bQLyUzWqbue9atqg/view)

**Team(s) Requesting:** CZI SC

---

**Related tickets:**

- [x] #[4608](https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/gh/chanzuckerberg/single-cell-data-portal/4608)

### CODE REVIEW

**Review by tihuan — APPROVED** [review]
Congrats on the first PR to SDS, @kaloster ! Really appreciate your contribution 🎉 !

**Review by masoudmanson — CHANGES_REQUESTED** [review]
Thank you so much for your contribution to the SDS. The SVG file only need a minor modification; otherwise, it LGTM. 🎉

**Comment by kaloster** [comment]
@masoudmanson sorry saw this too late - happy to push any changes to the SVG

File: packages/components/src/common/svgs/IconQuoteLarge.svg
Comment (masoudmanson) [inline packages/components/src/common/svgs/IconQuoteLarge.svg:1]: If you delete the fill attribute from the path tag, the icon will take on colors other than the Primary blue.
Currently, it only displays the blur color. 

Screenshot 2023-09-27 at 3 37 16 PM

Comment (kaloster) [reply packages/components/src/common/svgs/IconQuoteLarge.svg:1]: gotcha! on it

**Comment by masoudmanson** [comment]
> @masoudmanson sorry saw this too late - happy to push any changes to the SVG

That's okay. You can make a new PR and fix the SVG file. Thank you very much. 💯 🥳

**Pull request merged**